dash/include/dash/memory/GlobStaticMem.h

```diff
@@ -277,13 +277,14 @@ class GlobStaticMem
   /**
    * Copy-assignment operator.
    *
-   * \TODO make move constructor defaultable by using RAII 
    */
   self_t & operator=(const self_t & rhs)
     = delete;
 
   /**
    * Move-assignment operator.
+   *
+   * \TODO make move constructor defaultable by using RAII 
    */
   self_t & operator=(self_t && other) {
```

Comment:
Maybe I am missing something here: what is in the way of defaulting them? 

Also: the operator has no return statement.